oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyState.java

```diff
@@ -216,7 +264,7 @@ static PropertyState readProperty(String name, DocumentNodeStore store, JsopRead
     /**
      * Read a multi valued {@code PropertyState} from a {@link JsopReader}.
      *
-     * @param name the name of the property state
+     * @param name   the name of the property state
```

Comment:
Could you please remove the format changes.

Reply:
Done

Reply:
the revert for this wasn't done though?

Reply:
The revert was done.

Reply:
Done.